client/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "react-app-template",
+  "version": "1.0.0",
+  "description": "A template for react apps",
+  "main": "webpack.config.dev.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint --ext .js,.jsx,tsx src/",
```

Comment:
Looks like you forgot the decimal before tsx here, but will likely also want to include .ts files.

Reply:
Removed typescript.